### ISSUE TITLE
The plugin `disk_space` uses a deprecated version of the Android embedding

### ISSUE THREAD

**Issue by mr-mmmmore** [mboeddeker/disk_space#4]
When running `flutter pub get` with Flutter 2.5 and Dart 2.13 I get this warning message:

```
The plugin `disk_space` uses a deprecated version of the Android embedding.
To avoid unexpected runtime failures, or future build failures, try to see if this plugin supports the Android V2 embedding. Otherwise, consider removing it since a future release of Flutter will remove these deprecated APIs.
If you are plugin author, take a look at the docs for migrating the plugin to the V2 embedding: https://flutter.dev/go/android-plugin-migration.
```

I am using disk_space 0.1.1.

**Comment by juzejunior** [comment]
The same here.

**Comment by 1N50MN14** [comment]
@mboeddeker Any chance you could merge the PR? Many thanks.

**Comment by mr-mmmmore** [comment]
Hey @mboeddeker I see that you have merged the PR and upgraded the version to 0.2.1, but the new version is not published on pub.dev yet, are you going to do it soon?
